src/libraries/Common/src/System/Text/AsciiPolyfills.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text
+{
+    /// <summary>Provides downlevel polyfills for Ascii helper APIs.</summary>
+    internal static class Ascii
+    {
+        public static bool IsValid(string value)
```

Comment:
Why is this method needed. There is no corresponding `string` overload in the `System.Private.CoreLib`  implementation.

Reply:
Because on framework a `string` isn't implicitly convertible to `ReadOnlySpan<char>`, so callers need to use `IsValid(someString.AsSpan())`.
This lets them drop the `AsSpan()`.

Reply:
We _could_, but given that the intention here is to let us avoid requiring framework-specific logic in more places, I don't see why we would want to.

Reply:
Is there potential for using C# 14 extension operators in framework code?

Reply:
That's what https://github.com/dotnet/runtime/pull/114644 did.
I don't think we need it in any of these cases.

Reply:
Nevermind, user-defined implicit and explicit conversion extension operators are not available.